content/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph.md

```diff
@@ -73,7 +73,7 @@ The recommended formats explicitly define which versions are used for all direct
 | --- | --- | --- | ---|
 | Composer             | PHP           | `composer.lock` | `composer.json`, `composer.lock` |
 | `dotnet` CLI | .NET languages (C#, C++, F#, VB)  |   `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj` |  `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj`, `packages.config` |{% ifversion fpt or ghes > 3.1 %}
-| Go modules | Go | `go.mod` | `go.mod` |{% endif %}
+| Go modules | Go | `go.sum` | `go.mod`,`go.sum` |{% endif %}
```

Comment:
Thanks a lot for this change, @annarosenthal!

Is this change being backported to GHES 3.2? If not, we'll probably need to version GHES 3.2 to contain the existing content, and only apply the changes to dotcom and GHES 3.3 and up:

```suggestion
| `dotnet` CLI | .NET languages (C#, C++, F#, VB)  |   `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj` |  `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj`, `packages.config` |
{%- ifversion fpt or ghes > 3.2 %}
| Go modules | Go | `go.sum` | `go.mod`, `go.sum` |
{%- elsif ghes = 3.2 %}
| Go modules | Go | `go.mod` | `go.mod` |
{%- endif %}
```

Reply:
thanks for taking a look @lucascosti. I confirmed with @iamwillbar that we won't be backporting it so I'll commit your suggestion for review.